packages/@lwc/template-compiler/src/__tests__/fixtures/directive-inner-html/valid/expected.js

```diff
@@ -0,0 +1,46 @@
+import { registerTemplate, sanitizeHtmlContent } from "lwc";
+function tmpl($api, $cmp, $slotset, $ctx) {
+  const { h: api_element } = $api;
+  return [
+    api_element(
+      "div",
+      {
+        props: {
+          innerHTML:
+            $ctx._rawHtml$0 !== ($ctx._rawHtml$0 = "Hello <b>world</b>!")
+              ? ($ctx._sanitizedHtml$0 = sanitizeHtmlContent(
+                  "Hello <b>world</b>!"
+                ))
+              : $ctx._sanitizedHtml$0,
```

Comment:
This can be optimized as we know that the string will never change. That said it feels like a micro-optimization for a super edge case.

```js
$ctx._rawHtml$0 || ($ctx._rawHtml$0 = sanitizeHtmlContent("Hello <b>world</b>!"))
```

Reply:
Added https://github.com/salesforce/lwc/pull/2256/commits/d6b581e0401252981473c3947d4afc1e479c1f94, though i also feel that's an edge case. lmk if you want to revert.